packages/SketchMorph2-Tests.package/M2BackendTests.class/instance/multipleFillAreaTestUsingUndo.testing..st

```diff
@@ -0,0 +1,32 @@
+testblocks
+multipleFillAreaTestUsingUndo: anUndoBlock testing: aTestBlock
```

Comment:
What is the reason for such a complex test which includes tests for things not related to undo itself (like brush tool, fill area tool etc.). I think the test should be as small as possible to only test one specific detail.